pkg/server/helpers.go

```diff
@@ -352,3 +354,23 @@ func resolveSymbolicLink(path string) (string, error) {
 	}
 	return filepath.EvalSymlinks(path)
 }
+
+// LoadCgroup loads the cgroup associated with path if it exits. If the cgroup
+// is not created it is created and returned.
+func LoadCgroup(cgroupPath string) (cgroups.Cgroup, error) {
```

Comment:
s/ LoadCgroup loads the cgroup associated with path if it exits/ loadCgroup loads the cgroup associated with path if it exits, and move the current process into the cgroup.

nit: Make it private if we don't want to expose it.